content/en/about/day-in-life-dev.md

```diff
@@ -6,95 +6,111 @@ position: 7
 category: 'About Us'
```

Comment:
Ack, I was too aggressive with the auto-formatter.

Will point out non-formatting changes and where I'd like feedback.